packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -208,13 +208,28 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
     } = props;
     if (hasCloseButton) {
       return (
-        <IconButton
-          className={ css('ms-Panel-closeButton ms-PanelAction-close', styles.closeButton) }
-          onClick={ this._onPanelClick }
-          ariaLabel={ closeButtonAriaLabel }
-          data-is-visible={ true }
-          iconProps={ { iconName: 'Cancel' } }
-        />
+        <div className={ css('ms-Panel-navigation', styles.navigation) } >
+          <IconButton
+            styles={
+              {
+                root: {
+                  height: 'auto',
+                  width: '44px',
+                  color: ColorClassNames.neutralSecondary,
```

Comment:
not sure what colorclassnames does, but why not use Theme.palette.neutralSecondary?

Reply:
Ahh, true true. I just grabbed the first color related thing I could find :) Grabbing from theme makes more sense.